### PR TITLE
Feature edit seller information

### PR DESCRIPTION
**Pull request by Dustnclay**
# Description
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, but not tested (may need new tests)

# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

**Review by aaronmatson1 — APPROVED** [review]
Looks good!

File: src/api/index.js [hunk 1]
```diff
@@ -49,6 +52,30 @@ const getProfileData = authState => {
...
+    );
+  } catch (error) {
+    return new Promise(() => {
+      console.log(error);
+      return [];
+    });
+  }
+};
+
+const putProfileData = (url, editedData, authState) => {
```
Comment (edelveiss) [inline src/api/index.js]: You can rename this function to putData. So we could reuse putData for editing other resources.

**Review by edelveiss — COMMENTED** [review]
Great job, Dustin! Just rename putProfileData function.

**Review by edelveiss — APPROVED** [review]
Great!

**Pull request merged**